conda_smithy/configure_feedstock.py

```diff
@@ -384,7 +384,7 @@ def main(forge_file_directory):
     build_config.CONDA_PY = 10
 
     recipe_dir = 'recipe'
-    config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash'},
+    config = {'docker': {'image': 'condaforge/linux-anvil', 'command': 'bash -ex'},
```

Comment:
Though one thing we might want to tweak is to add a section disabling the expansion around things like the `BINSTAR_TOKEN`. We want to make sure that isn't ending up in the log.